content/docs/getting-started/scaffolding-components/index.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Scaffolding Components in Backstage
+lastUpdated: '2021-11-24T14:00:00.0Z'
+description: How to create components such as services or websites using the Backstage scaffolder.
+---
+
+## Introduction
+
+This tutorial will show you how to create a basic scaffolder template in Backstage.
+
+## Prerequisite 
```

Comment:
You have to enable it in the settings too right? An admin could turn off the scaffolder for a tenant, then another user of the same tenant could read this and be confused.

Reply:
The first step explains how to enable it. Are you suggesting that I move that up to the prerequisites?

Just not sure what you mean.

Reply:
Ah yes. Indeed it does!